ra/ra_test.go

```diff
@@ -688,12 +690,11 @@ func TestAuthzRateLimiting(t *testing.T) {
 	_, _, ra, fc, cleanUp := initAuthorities(t)
 	defer cleanUp()
 
-	ra.rlPolicies = ratelimit.RateLimitConfig{
-		PendingAuthorizationsPerAccount: ratelimit.RateLimitPolicy{
+	ra.rlPolicies.SetPendingAuthorizationsPerAccountPolicy(
```

Comment:
It looks like these setter methods are implemented and exported solely for the sake of testing. It would be more Go-like to not implement them, and instead use a mocked out RateLimitConfig for the RA test.